### ISSUE TITLE
Feature request / question: newline at the end of the paste text

### ISSUE THREAD

**Issue by tzach** [choldgraf/sphinx-copybutton#56]
When pasting a copy text, an extra newline is added at the end.
On the terminal, it means an Enter command which starts executing.
Any way to remove it?

**Comment by choldgraf** [comment]
Ah, that's a good point - I'd be +1 on this. I think that the fix would need to be here:

https://github.com/choldgraf/sphinx-copybutton/blob/master/sphinx_copybutton/_static/copybutton.js#L72

That is the code that grabs the text content of the cell and returns the text to be copied. You'd want to modify that to check for an empty final line and remove it

**Comment by tzach** [comment]
Indeed! 
The following change fixes the problem:

```
- return textContent.join('\n')
+ return textContent.join('')
```

**Comment by choldgraf** [comment]
Wouldn't that remove all the newlines *inside* the code as well? I think we only want to remove the very last newline

**Comment by tzach** [comment]
You are right, sorry my JS is rusty
This should do the work
```
return textContent.slice(0, -1).join('\n') + textContent.slice(-1);
```

**Comment by choldgraf** [comment]
this is fixed in #58
